scripts/build.ps1

```diff
@@ -570,7 +570,7 @@ function Create-VsixPackage
     $testPlatformExternalsVersion = ([xml](Get-Content $env:TP_ROOT_DIR\scripts\build\TestPlatform.Dependencies.props)).Project.PropertyGroup.TestPlatformExternalsVersion
 
     # Copy legacy dependencies
-    $legacyDir = Join-Path $env:TP_PACKAGES_DIR "Microsoft.Internal.TestPlatform.Extensions\$testPlatformExternalsVersion-patched3\contentFiles\any\any"
+    $legacyDir = Join-Path $env:TP_PACKAGES_DIR "Microsoft.Internal.TestPlatform.Extensions\$testPlatformExternalsVersion-patched5\contentFiles\any\any"
```

Comment:
@AbhitejJohn  We talked about this on call, just a reminder so this does not go unfixed. 🙂  Please base this package on patched4 package, because that includes the correct symbols for trace data collector.

Reply:
yup, already did so. I've deleted the old patched-5 to avoid confusion and pushed up an updated version based on patched-4.